Source/cmake/OptionsWPE.cmake

```diff
@@ -165,25 +165,16 @@ if (ENABLE_DOCUMENTATION AND NOT GIDocgen_FOUND)
 endif ()
 
 if (ENABLE_WPE_1_1_API)
-    set(SOUP_MINIMUM_VERSION 3.0.0)
-    set(SOUP_API_VERSION 3.0)
     set(WPE_API_VERSION 1.1)
     set(WPE_API_MAJOR_VERSION 1)
-    set(ENABLE_SERVER_PRECONNECT ON)
     SET_AND_EXPOSE_TO_BUILD(ENABLE_2022_GLIB_API OFF)
 else ()
-    set(SOUP_MINIMUM_VERSION 3.0.0)
-    set(SOUP_API_VERSION 3.0)
     set(WPE_API_VERSION 2.0)
     set(WPE_API_MAJOR_VERSION 2)
-    set(ENABLE_SERVER_PRECONNECT ON)
     SET_AND_EXPOSE_TO_BUILD(ENABLE_2022_GLIB_API ON)
 endif ()
-find_package(LibSoup ${SOUP_MINIMUM_VERSION} REQUIRED)
 
-if (NOT LibSoup_FOUND)
-    message(FATAL_ERROR "libsoup 3 is required.")
-endif ()
+find_package(LibSoup 3.0.0 REQUIRED)
```

Comment:
Should move this up to the top of the file.

Reply:
Done!